crates/utils/src/settings/structs.rs

```diff
@@ -63,6 +63,12 @@ pub struct DatabaseConfig {
   /// Maximum number of active sql connections
   #[default(5)]
   pub pool_size: usize,
+
+  /// Whether to use TLS. If enabled, uses the platform's native certificate store by default.
+  /// Optionally, a path to a CA certificate file can be provided with the LEMMY_DATABASE_CERT_PATH
+  /// env variable.
+  #[default(false)]
+  pub tls_enabled: bool,
```

Comment:
Never prefer env vars, most of which aren't supported. This is going to get so scatterbrained really quickly trying to support multiple undocumented env vars, rather than a single, well-defined config file... I regret merging every one of them. 

Make this an optional `tls_cert_path` that can be provided.

Reply:
Makes sense, I'll make some changes